website/src/services/emailService.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  welcomeEmailText,
+  welcomeEmailHTML,
+  welcomeEmailCSS,
+} from '@/services/welcomeEmailContent';
+
+/**
+ * Handles the logic for sending a welcome email.
+ * @param email - The recipient's email address.
+ */
+export async function handleSendWelcomeEmail(email: string): Promise<void> {
```

Comment:
**question:** Why not using a [server action](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations) here ? This would be make the code way easier to maintain ver time instead of splitting service / api